doc/index.rst

```diff
@@ -49,51 +49,8 @@ oriented interface or via a set of functions familiar to MATLAB users.
 Documentation
 -------------
 
-This is the documentation for Matplotlib version |version|.
```

Comment:
The section does not cover versions anymore. It's ok to remove this because we already have it more prominently in the title.
![grafik](https://user-images.githubusercontent.com/2836374/72669035-54de1000-3a2d-11ea-9eb0-f3a4524d99a1.png)